priv/repo/migrations/20220928131105_add_execution_result.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Wanda.Repo.Migrations.AddExecutionResult do
+  use Ecto.Migration
+
+  def change do
+    create table(:execution_results, primary_key: false) do
+      add :execution_id, :uuid, primary_key: true
+      add :group_id, :uuid, null: false
```

Comment:
What are these `null: false` for?

Reply:
It should mean that the field cannot be nullable

Reply:
Thanks Carmine, it's exactly that! At a database level.

In the following steps we will discover if we need some `validate_required` magic at the entity level.